auth/manage_users.js

```diff
@@ -35,6 +35,17 @@ admin.auth().getUserByPhoneNumber(phoneNumber)
   });
 // [END get_user_by_phone]
 
+// [START get_user_by_federated_id]
```

Comment:
These tags/comments don't seem to quite match up with the function name, so likely no action required. But technically, this function can look up by non-federated ids too. (Here and line 125.)

That said, we do want to discourage users from looking up non-federated ids with these functions as there are other more direct ways to do so. In particular, I like your comment on line 126 mentioning 'federated' and think it should be kept as is.